lib/workload/stateful/filemanager/filemanager/src/events/aws/collecter.rs

```diff
@@ -0,0 +1,246 @@
+#[double]
```

Comment:
This expands to:
```rust
#[cfg(not(test))]
use crate::clients::aws::s3::Client;
#[cfg(test)]
use crate::clients::aws::s3::MockClient as Foo;
```
Which allows using different versions of the client in testing code compared to the main code.